src/main/java/step02/code/domain/GradeEnum.java

```diff
@@ -1,31 +1,57 @@
 package step02.code.domain;
 
 import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public enum GradeEnum {
   FIRST(6, 2000000000),
-  SECOND(5, 1500000),
-  THRID(4, 50000),
-  FOURTH(3, 5000),
+  SECOND(5, 30000000), // 5 + 보너스 숫자 일치
```

Comment:
```suggestion
  SECOND(5, 30_000_000), // 5 + 보너스 숫자 일치
```
1,000 이상의 숫자를 표현할때는 가독성을 고려한 표현을 넣을 수 있습니다.